index.html

```diff
@@ -0,0 +1,234 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="container">
+        <div id="template1">
+            <header>
+                <a href="#" class="logo">My Logo</a>
+                <nav class="Union">
+                    <img src="images/header/Union.png" alt="bar">
+                </nav>
+            </header>
+            <div class="bg_image"></div>
+                <section id="landing_page">
+                    <div class="intro">
+                        <h1>I’m Praveen Glad to see you!</h1>
+                    </div>
+                    <div class="description">
+                        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    </div>
+                    <div class="connect">
+                        <h3>LET'S CONNECT</h3>
+                    </div>
+                    <ul>
+                        <li><a href=""><img class="twitter" src="images/header//twiter.png" alt="twiter_logo"></a></li>
+                        <li><a href=""><img class="linkedIn" src="images/header//linkedin.png" alt="linkedIn_logo"></a></li>
+                        <li><a href=""><img class="m" src="images/header//m.png" alt="m_logo"></a></li>
+                        <li><a href=""><img class="github" src="images/header//github.png" alt="github_logo"></a></li>
+                        <li><a href=""><img class="ok" src="images/header//ok.png" alt="ok_logo"></a></li>
+                    </ul>
+                </section>
+        </div>
+         <!-- work -->
+         <div id="work_1">
+            <div class="a">
+                <img src="images/header/tonic.png" alt="tonic">
+            </div>
+            <div class="b">Tonic</div>
```

Comment:
- [ ] Kindly update this project's title (and all the others) to an `hx` tag (`h2` or `h3`) because they are a header 👍 
- [ ] Kindly try to follow the Figma design as much as possible. Kindly update the project cards to make them look more like Figma, this includes the alignment of the cards (yours are on the left while they're supposed to be centered), the technologies badges, the `See project` button, and the borders of the card (kindly add them)

## Your design:
![image](https://user-images.githubusercontent.com/50721479/192097188-d3dda498-104c-410e-8333-b1a4ba4c599c.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/192097193-ee6bb075-d921-451c-9cd1-f06f47668e00.png)